config/settings.yaml.example

```diff
@@ -35,6 +35,10 @@
 :domain: 'localdomain.net'
 :fqdn: 'localhost.localdomain.net'
 
+# Add hosts to config.hosts here (fqdn is added automatically; no need to add it here)
```

Comment:
That's only true for development, not production
```suggestion
# Add hosts to config.hosts here
```

It may also make sense to link to upstream docs and refer to the explicit names
```suggestion
# Configure hostnames for ActionDispatch::HostAuthorization
# Only hostnames are supported. Regular expressions and IP addresses/ranges are not.
# https://guides.rubyonrails.org/v6.1/configuring.html#configuring-middleware
```